<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClazzTimeTableWidget</class>
 <widget class="QWidget" name="ClazzTimeTableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="8">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTableView" name="tableView">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QWidget" name="timeTableWidget" native="true"/>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QCheckBox" name="editableCheckBox">
     <property name="text">
      <string>可修改</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="saveButton">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
     <property name="class" stdset="0">
      <string>button</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="7">
    <widget class="QPushButton" name="exportAllTimeTableOfAllClazzesButton">
     <property name="text">
      <string>导出所有课表</string>
     </property>
     <property name="class" stdset="0">
      <string>button</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QPushButton" name="exportTimeTableOfClazzButton">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>导出当前课表</string>
     </property>
     <property name="class" stdset="0">
      <string>button</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="courseTeacherButton">
     <property name="text">
      <string>课程/教师选择窗口</string>
     </property>
     <property name="class" stdset="0">
      <string>button</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
